### DISCUSSION TITLE
Updraft Section 4 Lesson 40 - Written lesson incomplete

### DISCUSSION THREAD

**Discussion by rube-de** [Cyfrin/security-and-auditing-full-course-s23#103] [General]
The written lesson of Lesson 40 missing half of what is talked in the video lesson.
The first text is about low level calls which is half way through the video.

![image](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/8930910/05b59f5f-64f1-4362-975e-1d9a8b88d4b6)

**Comment by PatrickAlphaC** [comment]
Thanks! We will take a look cc @cromewar
